src/core/issues-notes.js

```diff
@@ -19,24 +19,25 @@ export const name = "core/issues-notes";
 const MAX_GITHUB_REQUESTS = 60;
 
 function handleIssues(ins, ghIssues, conf) {
-  const $ins = $(ins);
+  window.ins = ins;
+  window.ghIssues = ghIssues;
+  window.conf = conf;
   const { issueBase, githubAPI } = conf;
   const hasDataNum = !!document.querySelector(".issue[data-number]");
   let issueNum = 0;
-  const $issueSummary = $(
+  const issueSummary = htmlToElement(
     "<div><h2>" + conf.l10n.issue_summary + "</h2><ul></ul></div>"
   );
-  const $issueList = $issueSummary.find("ul");
-  $ins
-    .filter((i, issue) => issue.parentNode)
-    .each((i, inno) => {
-      const $inno = $(inno);
-      const isIssue = $inno.hasClass("issue");
-      const isWarning = $inno.hasClass("warning");
-      const isEdNote = $inno.hasClass("ednote");
-      const isFeatureAtRisk = $inno.hasClass("atrisk");
-      const isInline = $inno[0].localName === "span";
-      const dataNum = $inno.attr("data-number");
+  const issueList = issueSummary.querySelectorAll("ul")[0];
```

Comment:
We can use `querySelector` if we only need the first element.
```suggestion
  const issueList = issueSummary.querySelector("ul");
```